pkg/api/const.go

```diff
@@ -223,6 +223,8 @@ const (
 	DefaultMaximumLoadBalancerRuleCount = 250
 	// DefaultEnableAutomaticUpdates determines the aks-engine provided default for enabling automatic updates
 	DefaultEnableAutomaticUpdates = true
+	// DefaultEnableCSIProxyWindows determines if CSI proxy should be enabled by default for Windows nodes
+	DefaultEnableCSIProxyWindows = false
```

Comment:
you can move this to api/common/const.go if it'll be shared by api/vlabs

Reply:
oh that works - let me do that instead of what I just did

Reply:
I moved it to api/common/const.go